magento/magento-cloud-docker
# MCLOUD-5691: Add support for MySQL #189

### Description

Since Magento supports both databases, we should add support for MySQL alongside with MariaDB.

### Fixed Issues (if relevant)

1. #189

### Manual testing scenarios

1. Deploy Magento on Cloud Docker
2. Env works fine
3. Drop Cloud Docker environment
4. Build config with `build:compose --db 5.6 --db-image mysql`
5. Deploy environment
6. The environment works as expected

### Release notes

Added the support for the MySQL database. Now, you can configure the Cloud Docker environment with either MariaDB or MySQL.

### Associated documentation updates

https://github.com/magento/devdocs/pull/7356

### Contribution checklist
